uiowa/uiowa
# Started updating one--col--background layout template

Closes #2369 


# How to test


- Sync a site
- `blt frontend`
- Check a page with a featured image (or add a featured image) and a page without
  - Page without featured image: No visible changes should be present, particularly the page title/breadcrumbs should not be affected
  - Page with featured image: Featured image should be visible and appropriately styled, as well title/breadcrumbs. Sizing is different than previous hero implementation, but otherwise styling (image gradient, title positioning, etc) should be consistent
